src/Riok.Mapperly/Riok.Mapperly.csproj

```diff
@@ -17,8 +17,20 @@
     <PackageIcon>logo.png</PackageIcon>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <ROSLYN_VERSION Condition="'$(ROSLYN_VERSION)' == ''">4.4</ROSLYN_VERSION>
+    <DefineConstants Condition="'$(ROSLYN_VERSION)' == '4.4'">$(DefineConstants);ROSLYN4_0_OR_GREATER</DefineConstants>

```

Comment:
Shouldn't this constant be called `ROSLYN4_4_OR_GREATER`?

Reply:
Whooops, fixed.